model_hub/model_hub/utils.py

```diff
@@ -28,15 +28,15 @@ def expand_like(arrays: List[np.ndarray], fill: float = -100) -> np.ndarray:
     """
     full_shape = list(arrays[0].shape)
     if len(full_shape) == 1:
-        return np.concatenate(arrays)
+        return np.ndarray(np.concatenate(arrays))
     full_shape[0] = sum(a.shape[0] for a in arrays)
     full_shape[1] = max(a.shape[1] for a in arrays)
     result = np.full(full_shape, fill)
     row_offset = 0
     for a in arrays:
         result[row_offset : row_offset + a.shape[0], : a.shape[1]] = a
         row_offset += a.shape[0]
-    return result
+    return result  # type: ignore
```

Comment:
I'd decide on one of `type: ignore` or `np.ndarray(result)`. You're using both throughout this function.